grafana-plugin/src/pages/incidents/Incidents.tsx

```diff
@@ -649,6 +602,71 @@ class Incidents extends React.Component<IncidentsPageProps, IncidentsPageState>
     );
   }
 
+  shouldShowPagination() {
+    return this.state.pagination?.start && this.state.pagination?.end;
+  }
+
+  handleSelectedIncidentIdsChange = (ids: Array<Alert['pk']>) => {
+    this.setState({ selectedIncidentIds: ids }, () => {
+      ids.length > 0 ? this.clearPollingInterval() : this.setPollingInterval();
+    });
+  };
+
+  getTableColumns(): Array<{ width: string; title: string; key: string; render }> {
+    const { store } = this.props;
+
+    return [
+      {
+        width: '140px',
+        title: 'Status',
+        key: 'time',
+        render: withSkeleton(this.renderStatus),
```

Comment:
AFAIK we can get rid of `withSkeleton` fn, because  alert can't contain attribute `short` anymore, backend removed this